pkg/ebpf/events_derived.go

```diff
@@ -77,16 +79,19 @@ func (t *Tracee) deriveEvents(ctx context.Context, in <-chan *trace.Event) (<-ch
 				out <- event
 
 				// Derive event before parsing its arguments
-				derivatives, errors := events.Derive(*event, t.eventDerivations)
+				derivatives, errors := derive.Derive(*event, t.eventDerivations)
 
 				for _, err := range errors {
 					t.handleError(err)
 				}
 
 				for _, derivative := range derivatives {
+					// Drop derived events which were not requested by the user
```

Comment:
This is not needed anymore, the `Enabled` field in the derivation table takes care of that.